_locales/en/messages.json

```
@@ -35,6 +35,18 @@
         "message": "Manual Entry",
         "description": "Manual Entry."
     },
+    "migration_fail": {
+        "message": "Migrate data failed. You need export data from Google Authenticator again.",
```

Do we need all of these new strings? Failed could work here, otherwise use 
```suggestion
        "message": "Import failed. Please re-export your data from Google Authenticator and try again.",
```

---

I prefer the new string, because it guides users the next step.